app/models/linked_application_type.rb

```diff
@@ -1,9 +1,19 @@
 class LinkedApplicationType
-  LinkTypeStruct = Struct.new(:code, :description)
+  LinkTypeStruct = Struct.new(:code, :description, :label, :hint)
 
   LINK_TYPES = [
-    LinkTypeStruct.new(code: "FAMILY", description: "Family"),
-    LinkTypeStruct.new(code: "LEGAL", description: "Legal"),
+    LinkTypeStruct.new(
+      code: "FAMILY",
+      description: "Family",
+      label: I18n.t("providers.linking_case_confirmations.show.options.family_link.text"),
+      hint: I18n.t("providers.linking_case_confirmations.show.options.family_link.hint"),
+    ),
+    LinkTypeStruct.new(
+      code: "LEGAL",
+      description: "Legal",
+      label: I18n.t("providers.linking_case_confirmations.show.options.legal_link.text"),
+      hint: I18n.t("providers.linking_case_confirmations.show.options.legal_link.hint"),
```

Comment:
ideally these should not be in a model but located in the translations for views, to be more idiomatic